### PR TITLE
Added links and resources for Magento Cloud Docker support

### LINKED ISSUE #8066: Feedback on page: /cloud/docker/docker-config.html
# General issue

## Description:

Some merchants have been asking for assistance with their local Docker configuration - however, this is not supported by the Customer Engineering Team.

## Possible solutions:

This was previously [discussed with Margaret Eker in Slack](https://magento.slack.com/archives/C2KKBFDV4/p1578337546012500) about adding some sort of disclaimer in the page:

I don't recall exactly who had provided the following verbiage, but it should probably be included in some form on this page, which I've been providing to merchants who have requested Support assistance with Docker:

"Cloud Docker is not an Enterprise Magento Commerce Supported solution and is supported by the Magento Community 
The best path to get support for this would be to join the #Cloud-Docker Community Channel by visiting https://opensource.magento.com/slack-channels"

Affected page: 

- https://devdocs.magento.com/cloud/docker/docker-development.html

### PR DESCRIPTION
**Pull request by meker12**
Fixes #8066 

## Purpose of this pull request

Added Magento Community Engineering links and resources to get support when using Magento Cloud Docker for local development.

## Affected DevDocs pages

- https://devdocs.magento.com/cloud/docker/docker-development.html

### CODE REVIEW

File: src/cloud/docker/docker-development.md [hunk 1]
```diff
@@ -40,7 +40,23 @@ You can use the Docker environment to emulate the {{site.data.var.ece}} Integrat
 -  [Docker architecture and common commands]({{site.baseurl}}/cloud/docker/docker-containers.html)
 -  [Launch Docker development environment]({{site.baseurl}}/cloud/docker/docker-config.html)
 
+## Getting support for {{site.data.var.mcd-prod}}
+
+{{site.data.var.mcd-prod}} is a Magento Community Engineering project supported by the Magento developer community. You have several options to get support and learn more about {{site.data.var.mcd-prod}} and Magento local development.
```
Comment (hguthrie) [inline src/cloud/docker/docker-development.md:45]: It says Magento local development, but does it need to specify Cloud specifically, or is it okay to stay generic?

File: src/cloud/docker/docker-development.md [hunk 2]
```diff
@@ -40,7 +40,23 @@ You can use the Docker environment to emulate the {{site.data.var.ece}} Integrat
...
+
+-  **[Magento Community Engineering Slack organization]**–For support, questions, or discussion, chat with us in the **#cloud-docker** and **#cloud** channels. To join, send a request to _engcom@magento.com_ or [sign yourself up] using Slack.
+
+-  **[{{site.data.var.mcd-package}} GitHub repository]**–Visit the GitHub repository to read discussions about current issues, check current development, and submit issues or pull requests to improve the project.
+
+-  **Magento Cloud Community Engineering demos**–Magento hosts Cloud demo session where you can learn about developing Magento on the Cloud platform, including information about local development with {{site.data.var.mcd-prod}}}. For a schedule and recordings of previous demos, see the [Magento Cloud Community Engineering Demo Schedule].
```
Comment (hguthrie) [inline src/cloud/docker/docker-development.md:51]: Nothing to do here, but I did notice some strange order in their Spark page archive list.

**Review by hguthrie — APPROVED** [review]

**Review by YPyltiai — APPROVED** [review]

File: src/cloud/docker/docker-development.md (hunk 1 above)
Comment (meker12) [reply src/cloud/docker/docker-development.md:45]: Since you can use Docker for Cloud or on-premise Magento, I intentionally made it generic.

**Comment by meker12** [comment]
running tests

**Pull request merged**